pom.xml

```diff
@@ -22,62 +22,97 @@
         <postgresql.version>42.2.20</postgresql.version>
         <springfox.version>3.0.0</springfox.version>
         <spring.version>2.5.0</spring.version>
+        <jjwt.version>0.9.1</jjwt.version>
+        <jaxb-api.version>2.3.1</jaxb-api.version>
```

Comment:
never used

Reply:
<jaxb-api.version>2.3.1</jaxb-api.version> was deleted;
<jjwt.version>0.9.1</jjwt.version> is beign used;